CITATION.cff

```diff
@@ -0,0 +1,26 @@
+cff-version: 1.2.0
+title: 'nushell: a new type of shell'
+message: >-
+  If you use this software and wish to cite it, 
+  you can use the metadata from this file.
+type: software
+authors:
+  - family-names: The Nushell Project Developers
```

Comment:
```suggestion
  - name: "The Nushell Project Developers"
```

Reply:
`family-names` should be a "surname" for an individual.  For a "group/project", the citation format [uses `name`](https://github.com/citation-file-format/citation-file-format/blob/main/schema-guide.md#authors)

Reply:
Well caught!